apps/api/src/app/users/users.service.ts

```diff
@@ -0,0 +1,53 @@
+import {BadRequestException, Injectable, Logger, NotFoundException} from "@nestjs/common";
+import {InjectRepository} from "@nestjs/typeorm";
+import {UserModel} from "./model/user.model";
+import {Repository} from "typeorm";
```

Comment:
**suggestion:** Use prettier and always format your code, go to your Webstorm File -> Settings -> search "Prettier" -> select your prettier package -> on Format and Save -> Apply.

The select all your files in apps/api and do a Ctrl + Alt + L to reformat them